src/foam/nanos/auth/CapabilityAuthService.js

```diff
@@ -120,12 +142,15 @@ foam.CLASS({
         if ( x == null || permission == null ) return false;
         if ( x.get(Session.class) == null ) return false;
         if ( user == null || ! user.getEnabled() ) return false;
-
+        User agent = ((Subject) x.get("subject")).getRealUser();
```

Comment:
I don't think we should use the var name 'agent'. If we're using the user for an auth check, shouldn't we use getUser() instead?

Reply:
https://nanopay.atlassian.net/browse/NP-1371